schematools/contrib/django/factories.py

```diff
@@ -172,7 +169,12 @@ def __call__(
         kwargs = self.kwargs
         args = []
 
-        for modifier in self.modifiers:
```

Comment:
Is er een reden voor deze change? Dynamic handlers zijn makkelijk om mee te wekren.

Reply:
Ik zet de change terug. Beide werken